pyproject.toml

```diff
@@ -61,7 +60,7 @@ exclude = [
 namespaces = false
 
 [tool.setuptools.package-data]
-amical = [
+previs = [
     "data/eso_limits_matisse.json",
     "data/vizier_catalog_references.json"
 ]
```

Comment:
looks like this entire section was copied from AMICAL and should be dropped ?

Reply:
Already in the PR 👍

Reply:
looks to me that you just renamed the key